### DISCUSSION TITLE
Styles and Media doesnt apply to my website when published on github pages

### DISCUSSION THREAD

**Discussion by lostAlpaca88** [community/community#54863] [New to GitHub] [answered]
### Select Topic Area

Question

I am very new to github and web-development in general. My website looks completely different and honestly crappy when I view it using git hub pages. The stylesheet is not properly applied and media is completely invisible. no idea how to fix this

**Comment by iamreiyn** [comment]
Hi @lostAlpaca88 

I don't see any issues with your website or styling, perhaps it is fixed now?

I can see that you were using, for example, **./asset/bg.jpeg** to use an image but I think it should be **../asset/bg.jpeg** (notice the 2 dots before) as your "asset" folder is outside "docs" directory, hope it helps. Thank you!

**Comment by hrishikesh-kocharekar-dev** [comment]
Hi Syeda
It looks like your website is built using a static site generator called Jekyll, which is integrated with GitHub Pages. When you push changes to your repository, GitHub Pages automatically rebuilds your site using Jekyll and serves it to your visitors.

Based on the behavior you described, it sounds like there may be an issue with the way your CSS file is being loaded by Jekyll. Here are a few things you can check to try to resolve the issue:

Check that your CSS file is being loaded correctly: In your HTML file, make sure that the path to your CSS file is correct and that the file exists in the correct location in your repository. You can also try opening the CSS file directly in your browser to confirm that it's accessible.

Check that your CSS file is being processed by Jekyll: Jekyll automatically processes certain types of files, including CSS files, to apply any Liquid tags or other processing that may be necessary. Make sure that your CSS file has a `.css` extension and is located in the `_sass` or `assets/css` directory in your repository (these are the default directories that Jekyll looks for CSS files). You can also check the generated HTML source code to see if the CSS file is being included correctly.

Check that your site configuration is correct: Make sure that your `_config.yml` file in your repository is configured correctly for your site. This file contains settings that control how Jekyll processes your site, including the `baseurl` and `url` settings, which determine the root URL for your site.

If none of these steps resolve the issue, you may need to do some more troubleshooting to determine the cause of the problem. You can check the Jekyll documentation for more information on how to troubleshoot issues with Jekyll sites.

I hope this helps!

**Comment by lostAlpaca88** [comment, marked as answer]
its a lazy fix but i pasted all assets and css files in the same folder in which the index.html resided.

**Comment by Diegodotrivera** [comment]
I read somewhere if you push a commit with an empty title and in the comments type "rebuild" and push it, it seems to fix some issues with changes not updating.

Somehow this has worked form me every time when I've been sure my formatting and code was 100% correct. I hopw this helps
